test/dlc_tests/sanity/test_safety_check_v2.py

```diff
@@ -0,0 +1,99 @@
+import json
+import logging
+import sys
+
+import pytest
+
+from invoke import run
+from dataclasses import dataclass
+from typing import List
+
+from test.test_utils import (
+     is_canary_context
+)
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+LOGGER.addHandler(logging.StreamHandler(sys.stderr))
+
+SAFETY_FILE = '/var/safety_report.json'
+SAFETY_FILE_EXISTS = 0
+
+
+@dataclass
+class SafetyVulnerabilityAdvisory:
+    vid: str
+    advisory: str
+    reason_to_ignore: str
+    spec: str
+
+
+@dataclass
+class SafetyPackageVulnerabilityReport:
+    package: str
+    scan_status: str
+    installed: str
+    vulnerabilities: List[SafetyVulnerabilityAdvisory]
+    date: str
+
+    def __post_init__(self):
+        self.vulnerabilities = [SafetyVulnerabilityAdvisory(**i) for i in self.vulnerabilities]
+
+
+@dataclass
+class SafetyPythonEnvironmentVulnerabilityReport:
+    report: List[SafetyPackageVulnerabilityReport]
+
+    def __post_init__(self):
+        self.report = [SafetyPackageVulnerabilityReport(**i) for i in self.report]
+
+
+
+@pytest.mark.model("N/A")
+@pytest.mark.skipif(
```

Comment:
This is not necessary, because the test is currently not marked as canary, so it will not run anyway.